examples/secp256k1-transfer/src/lib.ts

```diff
@@ -0,0 +1,119 @@
+// payFeeByFeeRate
+import {
+  Indexer,
+  helpers,
+  Address,
+  Script,
+  RPC,
+  hd,
+  config,
+  Cell,
+  commons,
+  core,
+  WitnessArgs,
+  toolkit,
+  BI,
+} from "@ckb-lumos/lumos";
+import { BIish } from "@ckb-lumos/bi";
+import { values } from "@ckb-lumos/base";
+import { payFeeByFeeRate } from "@ckb-lumos/common-scripts/lib/common";
+const { ScriptValue } = values;
+
+const { AGGRON4 } = config.predefined;
+
+const CKB_RPC_URL = "https://testnet.ckb.dev/rpc";
+const CKB_INDEXER_URL = "https://testnet.ckb.dev/indexer";
+const rpc = new RPC(CKB_RPC_URL);
+const indexer = new Indexer(CKB_INDEXER_URL, CKB_RPC_URL);
+export const MIN_CELL_CAPACITY = BI.from(61 * 1e8);
+export type Account = {
+  lockScript: Script;
+  address: Address;
+  pubKey: string;
+};
+
+export const generateAccountFromPrivateKey = (privKey: string): Account => {
+  const pubKey = hd.key.privateToPublic(privKey);
+  const args = hd.key.publicKeyToBlake160(pubKey);
+  const template = AGGRON4.SCRIPTS["SECP256K1_BLAKE160"]!;
+  const lockScript = {
+    code_hash: template.CODE_HASH,
+    hash_type: template.HASH_TYPE,
+    args: args,
+  };
+  const address = helpers.encodeToAddress(lockScript, { config: AGGRON4 });
+  return {
+    lockScript,
+    address,
+    pubKey,
+  };
+};
+
+/**
+ * calculate the transaction fee by skeleton inputs and outputs
+ * @param skeleton unsigned transaction skeleton
+ * @returns the transaction fee, it is from `sum(skeleton inputs capacity) - sum(skeleton outputs capacity)`
+ */
+export function calculateTransactionFee(skeleton: helpers.TransactionSkeletonType) {
```

Comment:
`calculateTransactionFee` sounds like doing some calculation by tx size and fee rate, while the logic in this function is getting a paid tx fee, I suggest renaming it to `getPaidTxFee`